FluEgggui.m

```diff
@@ -374,53 +374,33 @@
             %% ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
             if strcmp(specie,'Silver')%if specie=='Silver'
                 %% STD
```

Comment:
Lori, can you please add a small line explaining how did you calculated this function and put your initials at the end of the comment??

Thanks.